### ISSUE TITLE
Docs version unknown

### ISSUE THREAD

**Issue by cdeil** [cta-observatory/ctapipe#705]
@kosack - Why is your docs version unknown?

![screen shot 2018-04-12 at 10 44 48](https://user-images.githubusercontent.com/852409/38666163-97bcb4cc-3e3e-11e8-9c43-31a187f00bc6.png)

**Comment by kosack** [comment]
that's strange!  I'm not sure - it used to work correctly. Hopefully it will start working again when the next tagged release is made (hopefully this week)

**Comment by kosack** [comment]
No, that was removed years ago, and it's worked fine since then.  We replaced it with a custom system that detects and sets the version based on the latest git tag.  On local copies the version gets set correctly, so it's something to do with the travis build. Not much has changed there, but it could be an updated dependency broke something. This was certainly working on the automatic docs as of a month ago. 

for example, i get:

```
ctapipe-info --version

*** ctapipe version info ***

version: 0.5.2.post509+git02986f2
```

And it works when I build the sphinx docs myself. 

![image](https://user-images.githubusercontent.com/11677812/38669050-a4a9830c-3e45-11e8-9af6-42e520a4dc67.png)
